ui/dashboards/src/context/DashboardAppSlice.tsx

```diff
@@ -27,7 +27,7 @@ export interface DashboardAppSlice {
   openPanelDrawer: (panelDrawer: PanelDrawer) => void;
   closePanelDrawer: () => void;
   panelGroupDialog?: PanelGroupDialog;
-  openPanelGroupDialog: (groupIndex: number) => void;
+  openPanelGroupDialog: (groupIndex?: number) => void;
```

Comment:
what's the use case for not specifiying the groupIndex (but calling openPanelGroupDialog())?

Reply:
when groupIndex is undefined, it means we're adding a new panel group